Header header

geometry_msgs/Vector3 gyroValues
geometry_msgs/Vector3 accValues
geometry_msgs/Vector3 magValues

